### DISCUSSION TITLE
Installing RDKit

### DISCUSSION THREAD

**Discussion by SG876** [rdkit/rdkit#4258] [Q&A]
I apologize for the basic question:  I have Anaconda installed on my Windows 10 machine.  In Spyder, I tried the installation instructions for RDKit (https://github.com/rdkit/rdkit/blob/master/Docs/Book/Install.md, "How to install RDKit with Conda"), but nothing.

Appreciate any help.  Thank you!

**Comment by greglandrum** [comment]
in order for us to be able to help, you will need to let us know exactly what you did and why you believe that things are not working.
The usual problem that people working with IDEs like Spyder have is that the IDE is using a different conda environment (or different python installation entirely) than the one in which they installed the RDKit.

**Reply by SG876** [reply]
I completely agree.  When I typed:

$ conda create -c conda-forge -n my-rdkit-env rdkit

Nothing happened.  It sort of just hung there.  So, I think it needs to have a channel set, or something similar.  I think it doesn't know where to look.  I was hoping to install the version on Github (b/c the version on Anaconda seems to be an outdated version:  anaconda.org/rdkit/rdkit).

Thank you for the help!

**Reply by greglandrum** [reply]
If you follow the instructions in the documentation it will install from conda-forge, which has the most recent released version of the RDKit. 

If conda install just freezes and doesn’t do anything, that’s not an RDKit problem or anything we can fix; you should probably search to see if you can find examples of someone else having problems like that (again, not RDKit specific)
